skins/oasis/css/core/PageHeader.scss

```diff
@@ -44,13 +48,11 @@
 		display: inline;
 		font-size: 26px;
```

Comment:
One thing I never got around to fixing was the alignment of the edit, talk, etc. buttons and pageshare links with the h1 in the header. It works fine for the max screenwidth, but will be slightly out for the next  breakpoint down as the font-size shrinks from 34px to 28px (26px is never used afaik). Any ideas how to fix it?